pangram.py

amymhaddad opened the pull request.

paulghaddad commented on pangram.py:
```diff
@@ -0,0 +1,21 @@
+    if sentence == '':
+        return False
+
+    for letter in sentence:
```
Nice job here!

paulghaddad commented on pangram.py:
```diff
@@ -0,0 +1,21 @@
+
+    for lowercase_letter in alpha_list_lowercase:
+        if lowercase_letter not in sentence_without_spaces:
+            return False
```
Need a line break to separate the `return True` as it is not part of the `for` loop.

paulghaddad commented on pangram.py:
```diff
@@ -0,0 +1,21 @@
+        if lowercase_letter != ' ':
+            sentence_without_spaces += lowercase_letter
+
+    for lowercase_letter in alpha_list_lowercase:
```
Really nice job iterating through all the lowercase letters! And immediately returning `False` and exiting the function if it's not in the sentence. You have your answer right then and there.

paulghaddad commented on pangram.py:
```diff
@@ -0,0 +1,21 @@
+def is_pangram(sentence):
+    """Is a sentence a pangram?"""
+
+    alpha_list_lowercase = string.ascii_lowercase
```
Consider making this a simpler name: `lowercase_letters`.

amymhaddad commented:
@paulghaddad I updated this exercise:
-Changed the variable name to the list of letters. However I have a question about this: shouldn't `lowercase_letters` be a constant? Pylint didn't say to make it a constant, but I think it should be. Thoughts?

-Added space at the end of the file for `Return True`
